recipes/python-toolchain/meta.yaml

```diff
@@ -0,0 +1,37 @@
+package:
+  name: python-toolchain
+  version: 1.0.0
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - python
+
+  run:
+    - python
+    - setuptools
+    - pip
+
+test:
+  commands:
+    - test -f "${PREFIX}/etc/pip.conf"                                 # [unix]
+   #- test -f "${STDLIB_DIR}/distutils/distutils.cfg"                  # [unix]
+    - if not exist "%LIBRARY_PREFIX%\\etc\\pip.conf" exit 1            # [win]
```

Comment:
I'm a little more worried about something parsing the yaml and misinterpreting something. As AppVeyor can handle this fine, I'm happy enough to leave this alone.